standard/tier/tier_utils.lua

```diff
@@ -0,0 +1,177 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Tier/Utils
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+-- module intended to be moved to `Module:Tier` after the old ones usage has been eliminated
+
+local HiddenSort = require('Module:HiddenSort')
+local Logic = require('Module:Logic')
+local Page = require('Module:Page')
+local String = require('Module:StringUtils')
+
+local TierData = mw.loadData('Module:Tier/Data')
+
+local NON_BREAKING_SPACE = '&nbsp;'
+
+local Tier = {}
+
+--- Converts input to standardized identifier format
+---@param input string|integer|nil
+---@return string|integer|nil
+function Tier.toIdentifier(input)
+	if String.isEmpty(input) then
+		return ''
```

Comment:
Is `''` a valid tier? If not then return nil.

Reply:
it is intended to be a fallback tier

Reply:
Maybe something more descriptive than `''`, like `'unknown'`?

And to clarify, `''` has a different meaning than `nil` right?

Reply:
the idea is to map nil to empty string via toIdentifier so that they are factually treated the same

fir commons that only has effect for tier but some wikis also have a default value for tier type